From 6bd56eadacfbecca869037726989d4b4b4c5a198 Mon Sep 17 00:00:00 2001
From: Olivier Moysan <olivier.moysan@foss.st.com>
Date: Thu, 8 Dec 2022 14:01:14 +0100
Subject: [PATCH] ARM: dts: stm32: enable timer on stm32mp157x-dk2 a7-examples

Enable timers examples that are normally kept disabled by default
on STM32MP157c-dk2 and STM32MP157f-dk2 boards.

Signed-off-by: Fabrice Gasnier <fabrice.gasnier@st.com>
Signed-off-by: Olivier Moysan <olivier.moysan@foss.st.com>
Change-Id: I6168616492f4dab8a200c295a81e55271c7cc0c3
---
 .../dts/st/stm32mp157c-dk2-a7-examples.dts    | 24 +++++++++++++++++++
 .../dts/st/stm32mp157f-dk2-a7-examples.dts    | 24 +++++++++++++++++++
 2 files changed, 48 insertions(+)

diff --git a/arch/arm/boot/dts/st/stm32mp157c-dk2-a7-examples.dts b/arch/arm/boot/dts/st/stm32mp157c-dk2-a7-examples.dts
index 4ab23646b124..bc70d75f3742 100644
--- a/arch/arm/boot/dts/st/stm32mp157c-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/st/stm32mp157c-dk2-a7-examples.dts
@@ -13,6 +13,30 @@ / {
 	compatible = "st,stm32mp157c-dk2-a7-examples", "st,stm32mp157c-dk2", "st,stm32mp157";
 };
 
+&timers1 {
+	status = "okay";
+};
+
+&timers3 {
+	status = "okay";
+};
+
+&timers4 {
+	status = "okay";
+};
+
+&timers5 {
+	status = "okay";
+};
+
+&timers6 {
+	status = "okay";
+};
+
+&timers12 {
+	status = "okay";
+};
+
 &uart7 {
 	status = "okay";
 };
diff --git a/arch/arm/boot/dts/st/stm32mp157f-dk2-a7-examples.dts b/arch/arm/boot/dts/st/stm32mp157f-dk2-a7-examples.dts
index c6493732c006..ae1cbefab958 100644
--- a/arch/arm/boot/dts/st/stm32mp157f-dk2-a7-examples.dts
+++ b/arch/arm/boot/dts/st/stm32mp157f-dk2-a7-examples.dts
@@ -13,6 +13,30 @@ / {
 	compatible = "st,stm32mp157f-dk2-a7-examples", "st,stm32mp157f-dk2", "st,stm32mp157";
 };
 
+&timers1 {
+	status = "okay";
+};
+
+&timers3 {
+	status = "okay";
+};
+
+&timers4 {
+	status = "okay";
+};
+
+&timers5 {
+	status = "okay";
+};
+
+&timers6 {
+	status = "okay";
+};
+
+&timers12 {
+	status = "okay";
+};
+
 &uart7 {
 	status = "okay";
 };
-- 
2.39.5

